ChatApp/Core/CustomUI/Sender/SenderMessageView.swift

```diff
@@ -49,8 +48,15 @@ final class SenderMessageView: UIView {
     }
     
     //MARK: - Internal Method
-    func setTextToBubble(_ text: String) {
-        messageLabel.text = text
+    func setTextToBubble(with message: MessageEntity) {
+        messageLabel.text = message.text
+        messageLabel.textColor = Constants.Color.messageTextBlackColor
+        dateLabel.text = message.isSent ? message.date?.stringRepresentation : Constant.errorMessage
+        dateLabel.textColor = Constants.Color.dateLabelTextGreyColor
+        if dateLabel.text == Constant.errorMessage {
+            messageLabel.textColor = Constants.Color.messageTextNoConnectionColor
+            dateLabel.textColor = Constants.Color.dateLabelErrorTextRedColor
+        }
```

Comment:
ლეიბლის კონტენტს რატომ ამოწმებ? 2 ხაზით ზემოთ მაინც შენი დასეტილია. მე ტერნერისაც მოვაშორებდი
```suggestion
        if !message.isSent {
            dateLabel.text = Constant.errorMessage
            messageLabel.textColor = Constants.Color.messageTextNoConnectionColor
            dateLabel.textColor = Constants.Color.dateLabelErrorTextRedColor
        } else {
             dateLabel.text = message.date?.stringRepresentation
             dateLabel.textColor = Constants.Color.dateLabelTextGreyColor
             messageLabel.textColor = Constants.Color.messageTextBlackColor
        }
```